resources/services/sql/sql_instances.go

```diff
@@ -115,9 +115,10 @@ func SQLInstances() *schema.Table {
 				Type:        schema.TypeBigInt,
 			},
 			{
-				Name:        "name",
-				Description: "Name of the Cloud SQL instance This does not include the project ID",
+				Name:        "id",
```

Comment:
`name` is actually an identifier for SQL instances according to the docs: https://cloud.google.com/sql/docs/mysql/instance-settings

Renaming `name` to `id` allows it to be included in the `gcp_resources` view (`id` field is required).

Reply:
I would keep both, as someone one reading  wouldn't expect this transformation